inginious/frontend/user_manager.py

```diff
@@ -121,6 +121,12 @@ def session_realname(self):
             return None
         return self._session.realname
 
+    def session_tos_signed(self):
+        """ Returns True if the current user has signed the tos"""
+        if not self.session_logged_in():
+            return None
+        return self._session.tos_signed
```

Comment:
It might be more secure to do a ``get("tos_signed", False)``. What happens if the user is logged in but hasn't accepted the ToS yet ? (in case of external login for instance)

Reply:
Any news ?